Automattic/wp-calypso
# Plans & FAQ: Improve Colour Contrast

#### Changes proposed in this Pull Request

This improves the plans contrast in three places by switching the variable to --text-subtle

1. The plans heading
<img width="219" alt="Screenshot 2019-08-05 at 21 09 01" src="https://user-images.githubusercontent.com/43215253/62491971-496f9300-b7c5-11e9-9d62-6b60a90eb55d.png">

2. The discount text in Jetpack Connect
<img width="904" alt="Screenshot 2019-08-05 at 21 07 59" src="https://user-images.githubusercontent.com/43215253/62491981-512f3780-b7c5-11e9-8a69-be956240ee76.png">

3. The FAQ headings on the Plans page
<img width="368" alt="Screenshot 2019-08-05 at 21 09 42" src="https://user-images.githubusercontent.com/43215253/62492005-5ab89f80-b7c5-11e9-903b-f8aa0429c277.png">


#### Testing instructions

Check the new colours and make sure the new variables are correctly assigned.

Fixes #35095
